docs/usage/misc.rst

```diff
@@ -0,0 +1,168 @@
+      verbose : bool, optional
+         Whether to print progress. Default is True.
+
+      Returns
+      -------
+      pdf_func : scipy.interpolate.interp1d
+         The PDF as a function.
+
+      Notes
+      -----
+      The PDF is saved as a scipy.interpolate.interp1d function with pickle.
+      """
+      pdf_x = []
+
+      xlim = (var.getMin(), var.getMax())
+      # Get a sample of x values
+      x = np.linspace(*xlim, npoints)
+
+      for k, i in enumerate(x, 1):
+         var.setVal(i)
+         # Evaluate the PDF at the given x value
+         pdf_x.append(pdf.getVal(var))
+         if verbose:
+               print(f"\t{path.replace('.pkl','')} progress [{k*100/len(x):.1f}%]", end="\r")
+      if verbose: print()
+
+      # Interpolate the PDF
+      pdf_func = interp1d(x, pdf_x)
+
+      with open(path, "wb") as f:
+         if verbose:
+            print(f"Saving model to {f.name}")
+         pickle.dump(pdf_func, f)
+
+      return pdf_func
+
+
+With zfit
+~~~~~~~~~
+
+This should be called after you have fitted your model and you have a ``zfit.pdf.BasePDF`` object.
+
+.. code-block:: python
+
+    from scipy.interpolate import interp1d
+    import pickle
+
+    def save_pdf(var, pdf, path="pdf.pkl", npoints=10000, verbose=True):
+        """
+        Save a PDF from zfit as a callable function.
+
+        Parameters
+        ----------
+        var : zfit.Space
+            The variable to evaluate the PDF at.
+        pdf : zfit.pdf.BasePDF
+            The PDF to save.
+        path : str, optional
+            The path to save the PDF to. Default is "pdf.pkl".
+        npoints : int, optional
+            The number of points to evaluate the PDF at. Default is 10000.
+        verbose : bool, optional
+            Whether to print progress. Default is True.
+
+        Returns
+        -------
+        pdf_func : scipy.interpolate.interp1d
+
+        Notes
+        -----
+        The PDF is saved as a scipy.interpolate.interp1d function with pickle.
+        """
+
+        lower, upper = var.limits
+        x = np.linspace(lower[-1][0], upper[0][0], npoints)
+
+        # Evaluate the PDF at the given points
+        pdf_x = zfit.run(pdf.pdf(x, norm_range=var))
+
+        # Interpolate the PDF
+        pdf_func = interp1d(x, pdf_x)
+
+        with open(path, "wb") as f:
+            if verbose:
+                print(f"Saving model to {f.name}")
+            pickle.dump(pdf_func, f)
+
+        return pdf_func
+
+
+Renormalize the PDF
+-------------------
+
+The ``pdf_func`` you get, by either get it from function or read the saved pickle file for ``RooFit`` or ``zfit``, has an area of 1. When you want to plot it, you need to multiply it by the bin width of your histogram, the number of expected events in the range and the integral of the PDF in the range. This can be done easily using this small function:
+
+.. code-block:: python
+
+   from scipy.integrate import quad
+
+   def renormalize(pdf, x_range, bins, n_data):
```

Comment:
Do we really need this two-step method (saving the pdf, then renormalising it)? Can't we do both in one function? Is it because we want the user to have two environments? One for fitting and the other one for plotting?

Reply:
Renormalization method is the same for `RooFit` and `zfit` once it's transformed to a `scipy` object. Usually, you want to save your pdfs with the integral equal to 1, and then renormalize only when plotting. The user can still do both in 1 function, but in the doc it's better to show this good practice. 
I agree with your last comment, but I finally don't think we need this function as an utility in `plothist`. In the doc it's enough.

Reply:
I don't think we necessarily want to scale to the number of data entries. Only the pdf of the total model should be scaled that way. So I think the function should take only 2 arguments: `pdf`, `scale_factor`. This also resolves my other comment, because in case of uneven binning, we let the user do the computation. I agree that this can stay outside of plothist. We can show how to compute the scaling factor outside of the function.

While I was writting this comment, I may start understanding better what we are doing: n_data is not the total number of data events in that range but the number of predicted data events in that range corresponding to a certain model component. If this is the case, the function can stay as is, but I would add this explanation in the documentation of the `n_data` argument and I would also mention in the documentation that a regular binning is assumed. In that regard, we should rename `bins` -> `n_bins`.

Reply:
You are right, `n_data` is indeed the number of predicted data events in that range corresponding to a certain model component.